<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookMediaEditor</class>
 <widget class="QDialog" name="BookMediaEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>تحرير الصور</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelDetails">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="pushRemoveMedia">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../share/moltaqa-lib/qt_rc.qrc">
             <normaloff>:/images/delete2.png</normaloff>:/images/delete2.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pushAddMedia">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../share/moltaqa-lib/qt_rc.qrc">
             <normaloff>:/images/add2.png</normaloff>:/images/add2.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../share/moltaqa-lib/qt_rc.qrc"/>
 </resources>
 <connections/>
</ui>
